src/pages/valvonta-kaytto/toimenpiteet.js

```diff
@@ -31,7 +31,10 @@ export const hasDeadline = R.propSatisfies(isDeadlineType, 'type-id');
 export const isCloseCase = isType(type.closed);
 export const isAuditCase = R.complement(isType(type.closed));
 export const isAuditCaseToimenpideType = R.propSatisfies(
-  R.includes(R.__, [1, 2, 3, 4, 5]),
+  R.includes(
+    R.__,
+    [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21]
```

Comment:
Is there anything like range in js to use here?

Reply:
On Ramdassa ja mietinkin sen käyttöä, mutta jätin toistaiseksi pois koska tästä voidaan haluta jättää joku välistä joskus tai muuta sellaista. Mutta ehkä laitan sen kuitenkin siihen, koska helppohan se on korvata myöhemmin muulla id-listauksella.